lib/shared/overlay_notification.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:async';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import 'package:overlay_support/overlay_support.dart';
+
+import 'package:globo_fitness/ressources/theme/theme_colors.dart';
+import 'package:globo_fitness/extensions/string_localized.dart';
+import 'package:globo_fitness/translations/locale_keys.g.dart';
+
+void showPopupNewConnectionStatus(bool isSuccess,
+        {bool shouldDismiss = true}) =>
+    isSuccess
+        ? (Timer.run(() => _showOverlay(
+            LocaleKeys.toast_message_success_connexion_message.localized(),
+            ThemeColors.skyBlue,
+            CupertinoIcons.check_mark_circled_solid,
+            ThemeColors.skyBlueOpacity,
+            shouldDismiss)))
+        : (Timer.run(() => _showOverlay(
+            LocaleKeys.toast_message_error_connexion_message.localized(),
+            ThemeColors.lightRed,
+            Icons.error_outline,
+            Colors.red,
+            shouldDismiss)));
+
+void _showOverlay(String message, Color color, IconData icon, Color iconColor,
+    bool shouldDismiss) {
```

Comment:
Add "{ }"  between the parameters for better comprehension when using the method :) 
( 
void _showOverlay({String message, Color color, IconData icon, Color iconColor,
     bool shouldDismiss = true} 
)